roles/module_openapi/templates/doc_generator.py

```diff
@@ -0,0 +1,913 @@
+from __future__ import absolute_import, division, print_function
```

Comment:
Not sure if templates is the right directory for this script. Can we move it under files?

Reply:
Moving forward, it might be better to make this an Action plugin.

Reply:
@NilashishC yea, moving all the logic to a smaller action logic is the aim here. Probably till then if we can keep it in template itself that'd be great.